projects/core/src/lib/util/components/not-found/not-found-page.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="container" fxLayout="row" fxLayoutAlign="center center">
+    <mat-card>
+        <mat-card-title>
+            <mat-icon mat-card-md-image svgIcon="geoengine:logo-alt" aria-label="Geo Engine" [inline]="true"></mat-icon>
+            <mat-card-title>404</mat-card-title>
+        </mat-card-title>
+        <mat-card-content>
+            <h2>This route is not on the map :(</h2>
+        </mat-card-content>
+
+        <mat-card-actions>
+            <button mat-button color="primary" (click)="goBack()">go back</button>
```

Comment:
"Go to map"

Reply:
[dddc37a](https://github.com/geo-engine/geoengine-ui/pull/512/commits/dddc37a3fc6800bc46a7f19f2071718cd9aa3f51)